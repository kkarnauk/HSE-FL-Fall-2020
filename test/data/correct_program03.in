f
. f.
f
. g f (ffdfjsh) sdas sda :- d. f :-
(g, g, g, ggg g g g); sdsada (sdasd) sdsad (sdasd); sfsdf, dfsdfsd (ssdf), (sdsf; sfs; fsdf; f)
.

f (((f))) :- f (ff f).
