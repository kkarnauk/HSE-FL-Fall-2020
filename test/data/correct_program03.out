|relation
|head
|	|identifier = 'f'

|relation
|head
|	|identifier = 'f'

|relation
|head
|	|identifier = 'f'

|relation
|	|head
|	|	|atom
|	|	|	|identifier = 'g'
|	|	|	|identifier = 'f'
|	|	|	|atom
|	|	|	|	|identifier = 'ffdfjsh'
|	|	|	|identifier = 'sdas'
|	|	|	|identifier = 'sda'
|	|body
|	|	|identifier = 'd'

|relation
|	|head
|	|	|identifier = 'f'
|	|body
|	|	|disjunction
|	|	|	|conjunction
|	|	|	|	|identifier = 'g'
|	|	|	|	|conjunction
|	|	|	|	|	|identifier = 'g'
|	|	|	|	|	|conjunction
|	|	|	|	|	|	|identifier = 'g'
|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|identifier = 'ggg'
|	|	|	|	|	|	|	|identifier = 'g'
|	|	|	|	|	|	|	|identifier = 'g'
|	|	|	|	|	|	|	|identifier = 'g'
|	|	|	|disjunction
|	|	|	|	|atom
|	|	|	|	|	|identifier = 'sdsada'
|	|	|	|	|	|atom
|	|	|	|	|	|	|identifier = 'sdasd'
|	|	|	|	|	|identifier = 'sdsad'
|	|	|	|	|	|atom
|	|	|	|	|	|	|identifier = 'sdasd'
|	|	|	|	|conjunction
|	|	|	|	|	|identifier = 'sfsdf'
|	|	|	|	|	|conjunction
|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|identifier = 'dfsdfsd'
|	|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|	|identifier = 'ssdf'
|	|	|	|	|	|	|disjunction
|	|	|	|	|	|	|	|identifier = 'sdsf'
|	|	|	|	|	|	|	|disjunction
|	|	|	|	|	|	|	|	|identifier = 'sfs'
|	|	|	|	|	|	|	|	|disjunction
|	|	|	|	|	|	|	|	|	|identifier = 'fsdf'
|	|	|	|	|	|	|	|	|	|identifier = 'f'

|relation
|	|head
|	|	|atom
|	|	|	|identifier = 'f'
|	|	|	|atom
|	|	|	|	|atom
|	|	|	|	|	|atom
|	|	|	|	|	|	|identifier = 'f'
|	|body
|	|	|atom
|	|	|	|identifier = 'f'
|	|	|	|atom
|	|	|	|	|identifier = 'ff'
|	|	|	|	|identifier = 'f'